core/model/src/main/java/org/activityinfo/model/expr/diagnostic/SymbolNotFoundException.java

```diff
@@ -1,9 +1,11 @@
 package org.activityinfo.model.expr.diagnostic;
 
 import org.activityinfo.model.expr.SymbolExpr;
+import org.activityinfo.model.formTree.FormTree;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.activityinfo.model.formTree.FormTree'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)